src/jenesis/tasks/monitor.py

```diff
@@ -77,7 +77,8 @@ def render(self):
                     progress_text = self._task_status_text[name]
 
             # render the status
-            print(f'  {glyph} {self._term.blue(name)}: {progress_text}'.ljust(self._term.width))
+            #print(f'  {glyph} {self._term.blue(name)}: {progress_text}'.ljust(self._term.width))
+            print(f'  {glyph} {self._term.blue(name)}: {progress_text}')
```

Comment:
I think that something like this might work to detect whether jenesis is running in an actual terminal or in a headless mode like in a github runner:
```suggestion
            width = self._term.width if self._term.is_a_tty else 80
            print(f'  {glyph} {self._term.blue(name)}: {progress_text}'.ljust(width))
```

Reply:
Ok thanks James, I will try it out